cmd/keymasterd/config.go

```diff
@@ -448,6 +452,31 @@ func loadVerifyConfigFile(configFilename string,
 		return nil, err
 	}
 
+	if hasDNS, err := runtimeState.Config.DnsLoadBalancer.HasDNS(); err != nil {
+		return nil, err
+	} else if hasDNS {
```

Comment:
There is no need for else. Also this function (loadVerifyConfigFile) is becoming too large, please create a new function to do everything in this branch.

Reply:
Split into new function. I'm not happy about expanding the if/else-if as it bloats the function by two lines.